src/pages/layout/header.css

```diff
@@ -3,20 +3,31 @@
 
 @value hover: #346;
 
+.header_bg {
+    background: #2F343C;
+}
+
 .container {
     composes: section from layout_css;
+    composes: header_bg;
     
-    flex: 0 0 180px;
+    flex: 1 1 360px;
+    min-width: 180px;
     
     z-index: 1;
     font-size: 0.8em;
     color: #FFF;
-    background: #2F343C;
+    background: header_background;
```

Comment:
Looks like this should be removed since you added `composes: header_bg` up above